Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:2.7.12
+
+ARG BUILD_DEPS=" \
+    build-essential \
+    git \
+    libffi-dev \
+    libssl-dev \
+    libxml2-dev \
+    libxslt1-dev"
+
+RUN apt-get update && \
+    apt-get install \
+        --no-install-suggests \
+        --no-install-recommends \
+        -y \
+        $BUILD_DEPS && \
+    rm -rf /var/lib/apt/lists/*
+
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
+
+RUN mkdir /code
+RUN mkdir /code/website
+RUN mkdir /code/plugins
```

Comment:
WORKDIR actually creates directories for you if they don't exist, so you can remove all three mkdirs here.

Only case you'd want to use mkdir beforehand is if you're running them under a different user, because WORKDIR will make them owned by root.